portality/static/widget/fixed_query_body_dev_edges.html

```diff
@@ -0,0 +1,68 @@
+<script type="text/javascript" src="/static/vendor/jquery-3.4.1/jquery-3.4.1.min.js"></script>
```

Comment:
Just a reminder for the cleanup: we should either put this code back in the original `fixed_query_body_dev.html` file, or remove the old files that we no longer need.

Reply:
Sure, it will be done. Kept so far as a reference.